doc/introduction/interfaces/numpy.rst

```diff
@@ -122,12 +122,12 @@ Differentiable and non-differentiable arguments
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 How does PennyLane know which arguments of a quantum function to differentiate, and which to ignore?
-For example, you may want to pass arguments as positional arguments to a QNode but *not* have
+For example, you may want to pass arguments to a QNode but *not* have
```

Comment:
I decided to polish the entire paragraph, so it makes sense together. I think here `as positional arguments` is confusing because we only state the rule about positional arguments below.